microservices-orders/scripts/render-connector-config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Why write the file to the system instead of just passing it in?

Reply:
There are three tools (curl, ccloud, and confluent local) that need to 'slurp' these values.  All of which do support direct slurping from files.  I found writing and reading files was the most straightforward way to template and pass the values to the various tools without having to deal with things like escaping values, dealing with shell issues, etc..  If you have have proposed changes that are more straightforward I'm very open to a suggested change here.

Reply:
Would this work? https://github.com/confluentinc/examples/blob/5.5.0-post/utils/ccloud_library.sh#L506-L509

Reply:
After more thought, I still prefer the files.  As an artifcat of running the demo it's something that we could point to for educational material and be referenced later by the demo runner.  Keeping this script independent of the tool that slurps the values seems helpful IMO.  It also may prove useful to have the file allowing a user to more easily run the command themselves directly (ccloud connector config -- -d file.config) after they run the demo.

Reply:
Ok let's try it out ;)